src/components/WwwFrame/TheHeader.vue

```diff
@@ -71,15 +71,15 @@
 									tw-rounded-full tw-overflow-hidden tw-object-fill"
 							>
 						</router-link>
-						<router-link
+						<kv-button
+							variant="secondary"
+							class="header__login__button"
 							:to="loginUrl"
-							v-show="isVisitor"
```

Comment:
This conditional is what swaps visibility of this button once the lender is logged in. Without it, I think this button would show up all the time...

Reply:
Oh, good catch. Didn't notice I removed it!